src/index.ts

```diff
@@ -195,6 +210,14 @@ export class UnleashClient extends TinyEmitter {
         this.context = { appName, environment, ...context };
         this.usePOSTrequests = usePOSTrequests;
         this.sdkState = 'initializing';
+        this.experimental = { ...experimental };
+
+        if (experimental?.togglesStorageTTL && experimental?.togglesStorageTTL > 0) {
```

Comment:
Pourrait être raccourci normalement
```suggestion
        if (experimental?.togglesStorageTTL > 0) {
```

Reply:
nop ça ne luit plait pas car togglesStorageTTL peut etre undefined

Reply:
`undefined` n'est pas supérieur à zéro normalement

Reply:
vu ensemble, on met en place une methode